src/main/flight/feedforward.c

```diff
@@ -153,6 +154,10 @@ FAST_CODE_NOINLINE float feedforwardApply(int axis, bool newRcFrame, feedforward
         } else {
             setpointDelta[axis] = setpointDeltaImpl[axis];
         }
+
+        // apply feedforward transition
+        setpointDelta[axis] *= ffTransitionFactor > 0 ? MIN(1.f, getRcDeflectionAbs(axis) * ffTransitionFactor) : 1;
```

Comment:
This line makes the pidControllerTest.testFeedForward in pid_unittest failed

Reply:
Thanks DusKing
I couldn't figure out why the change caused it to fail - not clever enough!
The test was expecting zero and it got an FF value, hence the failure.
But in actual testing the outcome was normal.
So I was not able to resolve this problem.

Reply:
I'm guessing that I need to include `feedforward.c` in the pid unit test?

Reply:
maybe @mikeller ?